integration-tests/api/__tests__/product-categories/materialized-path.ts

```diff
@@ -0,0 +1,74 @@
+import path from "path"
+import { ProductCategory } from "@medusajs/medusa"
+import { initDb, useDb } from "../../../helpers/use-db"
+
+describe("Product Categories > Tree Queries > Materialized Paths", () => {
```

Comment:
suggestion(non-blocking, minuscule): should we split this into several nested describes since when the feature is expanded more and additional integration tests are written this should be split up as well? 
(My thought was something like having the parent be "Product Categories", and then maybe having a "Tree Queries" in that)